### ISSUE TITLE
H91 test is refering to a "table above" that is missing

### ISSUE THREAD

**Issue by ShadowBB** [w3c/wcag#4153]
The [H91 technique test section](https://www.w3.org/WAI/WCAG22/Techniques/html/H91#tests) is referring to a "table above" in step 2 of the procedure.

This table seems to be missing to me.

**Comment by mbgower** [comment]
Thanks for opening this.
In the [technique for WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/H91.html), and the [WCAG 2.1 version](https://www.w3.org/WAI/WCAG21/Techniques/html/H91), a large table of HTML elements displayed before the examples. The table is no longer in the 2.2 version.

I suspect this may have been lost during a migration. I'm flagging @kfranqueiro to investigate.

**Comment by kfranqueiro** [comment]
The table appears to have been completely removed as part of #3757. It sounds like the removal was intentional: https://github.com/w3c/wcag/pull/3757/commits/d761be9ee6b9560025b8990f868ce68aa181dcca

Reassigning to @fstrr as that was his PR. Maybe the correct course of action here is to clean up the remaining reference to "table above"?
